src/main/resources/lang/polish.lang

```diff
@@ -0,0 +1,189 @@
+	finished loading: Załadowano %s skrypt¦¦ów¦ z %s trigger¦em¦ami¦ i %s komend¦ą¦ami¦ w %s
+
+# -- Skript command --
+skript command:
+	usage: Usage:
+	help:
+		description: Główna komenda Skript'a.
+		help: Wyświetla tę stronę pomocy. Użyj '/skript reload/enable/disable/update' aby otrzymać więcej informacji
+		reload:
+			description: Odświeża konfigurację, wszystkie skrypty, wszystko, albo wybrany skrypt
+			all: Odświeża konfigurację, wszystkie konfiguracje aliasów i wszystkie skrypty
+			config: Odświeża główną konfigurację
+			aliases: Odświeża konfigurację aliasów (aliases-*.sk lub JAR Skript'a)
+			scripts: Odświeża wszystkie skrypty
+			<script>: Odświeża wybrany skrypt lub folder skryptów
+		enable:
+			description: Włącza wszystkie skrypty lub wybrany skrypt
+			all: Włącza wszystkie skrypty
+			<script>: Włącza wybrany skrypt lub folder skryptów
+		disable:
+			description: Wyłącza wszystkie skrypty lub wybrany skrypt
+			all: Wyłącza wszystkie skrypty
+			<script>: Wyłącza wybrany skrypt lub folder skryptów
+		update:
+			description: Sprawdź aktualizację, przeczytaj listę zmian lub pobierz najnowszą wersję Skript'a
+			check: Sprawdza obecność najnowszej wersji
+			changes: Wypisuje wszystkie zmiany obecnej wersji
+			download: Pobiera najnowszą stabilną wersję
+		info: Wyświetla linki do aliasów i oficjalnej dokumentacji Skript'a
+		gen-docs: Generuje dokumentację używając szablonów w folderze Skript'a
+		test: Używane do testowania skryptów
+
+	invalid script: Nie znaleziono skryptu <grey>'<gold>%s<grey>'<red> w folderze skryptów!
+	invalid folder: Nie znaleziono folderu <grey>'<gold>%s<grey>'<red> w głównym folderze skryptów!
+	reload:
+		warning line info: <gold><bold>Linia %s:<gray> (%s)\n
+		error line info: <light red><bold>Linia %s:<gray> (%s)\n
+		reloading: Przeładowuję <gold>%s<reset>...
+		reloaded: <lime>Pomyślnie przeładowano <gold>%s<lime>. <gray>(<gold>%2$sms<gray>)
+		error: <light red>Znaleziono <gold>%2$s <light red>bł¦ąd¦ędów¦ podczas przeładowywania <gold>%1$s<light red>! <gray>(<gold>%3$sms<gray>)
+		script disabled: <gold>%s<reset> jest obecnie wyłączony. Użyj <gray>/<gold>skript <cyan>enable <red>%s<reset> aby go ponownie włączyć.
+		warning details: <yellow>    %s\n
+		error details: <light red>    %s\n
+		other details: <white>    %s\n
+		line details: <gold>    Linia: <gray>%s\n <reset>
+
+		config, aliases and scripts: konfiguracja, aliasy i wszystkie skrypty
+		scripts: wszystkie skrypty
+		main config: główna konfiguracja
+		aliases: aliasy
+		script: <gold>%s<reset>
+		scripts in folder: wszystkie skrypty w <gold>%s<reset>
+		x scripts in folder: <gold>%2$s <reset>skrypt¦¦ów¦ w <gold>%1$s<reset>
+		empty folder: <gold>%s<reset> nie ma w sobie żadnych włączonych skryptów.
+	enable:
+		all:
+			enabling: Włączanie wszystkich wyłączonych skryptów...
+			enabled: Pomyślnie włączono & zinterpretowano wszystkie wyłączone skrypty.
```

Comment:
```suggestion
			enabled: Pomyślnie włączono i zinterpretowano wszystkie wyłączone skrypty.
```
Argumentowałbym że i po angielsku lepiej by tu zwykłe `and` pasowało, ale po polsku tym bardziej `&` jest niespotykaną wstawką